tools/checkstyle/checkstyle.sh

```diff
@@ -53,6 +54,9 @@ run GoDeps \
 run GoFormat \
   "$GOFMT_PATH" -d .
 
+run GoImports \
+  "$GOIMPORTS_PATH" -d .
```

Comment:
Nit: https://pkg.go.dev/golang.org/x/tools/cmd/goimports

> In addition to fixing imports, goimports also formats your code in the same style as gofmt so it can be used as a replacement for your editor's gofmt-on-save hook.

So we can replace GoFormat with GoImports

Reply:
I want to test it, but yes, good point!